releases/v0.9.0-alpha0.md

```diff
@@ -0,0 +1,78 @@
+    * Resources can be dynamically delivered in confidential environments.
+    * Trustee is integrated into the Kata Containers CI.
+* All platforms now share one confidential rootfs.
+* All platforms share one confidential guest kernel.
+* Image request timeout is configurable to facilitate pulling large images.
+* Attestation Agent now supports generic `configfs-tsm` ABI for collecting evidence.
+* Enclave-cc moves to unified LibOS bundle for secure rootfs key handling and to the latest Occlum v0.30.1 release that adds SGX EDMM support for dynamically adjusting the enclave size.
+
+## Hardware Support
+
+This release does not officially support any hardware platforms.
+It is mainly intended for testing in non-tee environments.
+Future releases will return to previous levels of support.
+
+## Limitations
+
+The following are known limitations of this release:
+
+* Nydus snapshotter support is not mature.
+  * Nydus snapshot sometimes conflicts with existing node configuration.
+  * You may need to remove existing container images/snapshots before installing Nydus snapshotter.
+  * Nydus snapshotter may not support pulling one image with multiple runtime handler annotations even across different pods.
+    * These limitations can apply to the pause image when filesystem passthrough is not enabled.
+  * Host pulling with Nydus snapshotter is not yet enabled.
+  * Nydus snapshotter is not supported with enclave-cc.
+* Pulling container images inside guest may have negative performance implications including greater resource usage and slower startup.
+* `crio` support is still evolving.
+* Platform support is rapidly changing
+* SELinux is not supported on the host and must be set to permissive if in use.
+* The format of encrypted container images is still subject to change
+  * The [oci-crypt](https://github.com/containers/ocicrypt) container image format itself may still change
+  * The tools to generate images are not in their final form
+  * The image format itself is subject to change in upcoming releases
+  * Not all image repositories support encrypted container images.
+* Complete integration with Kubernetes is still in progress.
+  * OpenShift support is not yet complete.
+  * Existing APIs do not fully support the CoCo security and threat model. [More info](https://github.com/confidential-containers/community/issues/53)
+  * Some commands accessing confidential data, such as `kubectl exec`, may either fail to work, or incorrectly expose information to the host
+* The CoCo community aspires to adopting open source security best practices, but not all practices are adopted yet.
+  * We track our status with the OpenSSF Best Practices Badge, which improved to 69% at the time of this release.
+  * Vulnerability reporting mechanisms still need to be created. Public github issues are still appropriate for this release until private reporting is established.
```

Comment:
```suggestion
  * We resolved vulnerability reporting mechanisms and a security policy. Remaining work is to apply and document static and dynamic analysis. 
```

Reply:
I added something to the what's new section for this as well.